### PR TITLE
revise relative path and add a translated file installation_source.md

### PR DESCRIPTION
**Pull request by XiangruiCAI**
I find some pictures don't appear in the website. So I change the relative paths and arrange files like jp and kr.

I translated the installation_source page.

### CODE REVIEW

File: content/markdown/docs/zh/installation_source.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+* Q9: 当我安装 protocol buffer 时，被告知 GLIBC++_3.4.20 not found in /usr/lib64/libstdc++.so.6.
+
+  A9: 这说明链接器找到了 libstdc++.so.6，但是这个文件比用于编译和链接程序的GCC版本老。程序要求属于新版本GCC的libstdc++，所以必须告诉链接器怎么找到新版本的额libstdc++共享库。最简单的解决方法是找到正确的 libstdc++，并把它导出到 LD_LIBRARY_PATH 中。如, 如果GLIBC++_3.4.20 被下面的命令列出，
+
+      $ strings /usr/local/lib64/libstdc++.so.6|grep GLIBC++
+
+  你只需这样设置你的环境变量：
+
+      $ export LD_LIBRARY_PATH=/usr/local/lib64:$LD_LIBRARY_PATH
```
Comment (nudles) [inline content/markdown/docs/zh/installation_source.md:216]: There should be a A10.
Please rebase to the latest master.

Comment (XiangruiCAI) [inline content/markdown/docs/zh/installation_source.md:216]: A10 added.

File: content/markdown/docs/zh/overview.md [hunk 2]
```diff
@@ -19,23 +19,23 @@ SINGA的编程模型是基于层的抽象而设计，这对于深度学习模型
...
 ## 作业提交
 
-在SINGA中提交一个作业（即训练一个深度学习模型），用户需要将任务配置传给[主函数](../programming-guide.md)中的SINGA driver。作业配置需要明确图2中的四个主要部分：
+在SINGA中提交一个作业（即训练一个深度学习模型），用户需要将任务配置传给[主函数](programming-guide.md)中的SINGA driver。作业配置需要明确图2中的四个主要部分：
 
-  * [NeuralNet](../neural-net.md) ：描述神经网络结构，包括每层的具体设置和层与层的连接关系；
-  * [TrainOneBatch](../train-one-batch.md) ：该算法需要根据不同的模型类别而定制;
-  * [Updater](../updater.md) ：定义服务器端更新参数的协议；
-  * [Cluster Topology](../distributed-training.md) ：指定服务器和工作者的分布式拓扑架构。
+  * [NeuralNet](neural-net.md) ：描述神经网络结构，包括每层的具体设置和层与层的连接关系；
```
Comment (nudles) [inline content/markdown/docs/zh/overview.md]: Pls check the link format.
It should not be `[xxx](yyy.md)`
It should be `[xxx](yyy.html)`

Do not use the github pages to check the correctness.
You can compile the source files to generate html files by

```
 mvn site
```

Then you can check the imgs/link by opening the html files in target folder.

Comment (XiangruiCAI) [inline content/markdown/docs/zh/overview.md]: Sorry for these mistakes. I have changed the link format.

**Pull request merged**